CHANGELOG.md

```diff
@@ -1,22 +1,26 @@
 # Release History
 
-## 1.3.0 (TBD)
+## 1.3.0 (2023-03-28)
 
 ### New Features
 
 - Added support for `delimiters` parameter in `functions.initcap()`.
 - Added support for `functions.hash()` to accept a variable number of input expressions.
-- Added API `Session.RuntimeConfig` for getting/setting/checking the mutability of any runtime configuration.
+- Added API `Session.conf` for getting/setting/checking the mutability of any runtime configuration.
 - Added support managing case sensitivity in `Row` results from `DataFrame.collect` using `case_sensitive` parameter.
+- Added indexer support for `snowflake.snowpark.types.StructType`.
+- Added a keyword argument `log_on_exception` to `Dataframe.collect` and `Dataframe.collect_no_wait` to optionally disable error logging for sql exceptions.
 
 ### Bug Fixes
 
 - Fixed a bug where a DataFrame set operation(`DataFrame.substract`, `DataFrame.union`, etc.) being called after another DataFrame set operation and `DataFrame.select` or `DataFrame.with_column` throws an exception.
+- Fixed a bug where chained sort statement is overwritten by SQL simplifier.
```

Comment:
Does this sound a bit better?

"Fixed a bug where chained sort statements are overwritten by the SQL simplifier."